opentracing-web-servlet-filter/src/main/java/io/opentracing/contrib/web/servlet/filter/SpanWrapper.java

```diff
@@ -18,14 +18,24 @@
  */
 public class SpanWrapper {
 
-    private SpanManager.ManagedSpan managedSpan;
-    private AtomicBoolean finished = new AtomicBoolean();
+    private final SpanManager.ManagedSpan managedSpan;
+    private final AtomicBoolean finished = new AtomicBoolean();
 
+    /**
+     * @param span server span or null when request is not being traced
+     */
     protected SpanWrapper(Span span) {
         managedSpan = DefaultSpanManager.getInstance().activate(span);
     }
 
     /**
+     * @return whether request is being traced or not
+     */
+    public boolean isTraced() {
+        return managedSpan != null;
```

Comment:
The `activate` method always returns a `ManagedSpan` - so this would always return true?